tests/v2_validation/cattlevalidationtest/core/test_ebs_volume.py

```diff
@@ -0,0 +1,164 @@
+from common_fixtures import *  # NOQA
+
+if_test_ebs = pytest.mark.skipif(
+    RANCHER_EBS != "true",
+    reason="rancher ebs test environment is not enabled"
+)
+
+
+@if_test_ebs
+def test_environment_ebs_volume(client, super_client):
+    volume_name = "ebs_" + random_str()
+    volume = client.create_volume({"type": "volume", "driver": "rancher-ebs",
+                                   "name": volume_name, "driverOpts": {"size": "1"}})
+    launch_config = {"volumeDriver": "rancher-ebs",
+                     "dataVolumes": [volume_name + ":/test"],
+                     "networkMode": "managed",
+                     "imageUuid": "docker:ubuntu:14.04.3",
+                     "stdinOpen": True
+                     }
+    scale = 2
+
+    service, stack = create_env_and_svc(client, launch_config,
+                                        scale)
+    stack = stack.activateservices()
+    service = client.wait_success(service, 300)
+    assert service.state == "active"
+    container_list = get_service_container_list(client, service)
+    assert container_list[0].hostId == container_list[1].hostId
+
+    client.wait_success(client.delete(stack))
+    while True:
+        try:
+            client.delete(volume)
+            break
+        except Exception:
+            time.sleep(1)
+            pass
+
+
```

Comment:
Can we add a step to check if we can write to the volume and read from it ?
Also a short comment on the purpose of the test case does would be helpful.